act-rules-format.bs

```diff
@@ -101,132 +105,150 @@ An ACT Rule MUST have a unique identifier that can be any unique text, such as p
 
 Each [=accessibility requirement=] in the mapping MUST include the following:
 
-1. the name, title or summary of the accessibility requirement, and
+1. either the name, title, identifier or summary of the accessibility requirement, and
 2. the name of the [=accessibility requirements document=], and
-3. a link to the [=accessibility requirements document=] if one exists, and
+3. a link or reference to the [=accessibility requirements document=] if one exists, and
 4. the conformance level associated with the accessibility requirement, if one exists.
 
-For each accessibility requirement in the mapping, an ACT Rule MUST indicate what the [=outcome=] of the rule means for satisfying that accessibility requirement. When the outcome is `failed`, the accessibility requirements are *<dfn>not satisfied</dfn>*. When the outcome is `passed` or `inapplicable`, the accessibility requirements could be *<dfn>satisfied</dfn>*, or *<dfn>further testing could be necessary</dfn>*. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
+
+### Outcome Mapping ### {#outcome-mapping}
+
+For each accessibility requirement in the mapping, an ACT Rule MUST indicate what the [=outcomes=] of the rule means for satisfying that accessibility requirement for that test subject. When one or more of the [=outcomes=] for a test target is `failed`, the accessibility requirements are <dfn>not satisfied</dfn> for the test subject. When all of the [=outcomes=] are `passed` or `inapplicable`, the accessibility requirements could be <dfn>satisfied</dfn>, or <dfn>further testing could be necessary</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
 
 <div class=note>
-**Note**: In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
+  <p>**Note:** In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).</p>
 </div>
```

Comment:
Editorial: 
I think there should be comma after If a success criterion is *not satisfied* 
Suggested change: If a success criterion is *not satisfied*, a web page